cpp/test/neighbors/ann_cagra.cuh

```diff
@@ -497,9 +497,13 @@ class AnnCagraFilterTest : public ::testing::TestWithParam<AnnCagraInputs> {
         search_params.algo         = ps.algo;
         search_params.max_queries  = ps.max_queries;
         search_params.team_size    = ps.team_size;
-        search_params.itopk_size   = ps.itopk_size;
```

Comment:
You've reverted the change here and at line 616, but not at line 262. Is that intended?

Reply:
yeah, this was intentional.

When I was testing locally, I was only seeing a problem with the Cagra filter tests (line 616/line 600) - but not with the main unfiltered AnnCagraTest on line 262.  I wanted to maintain the test for k>1024 in the AnnCagraTest code, which requires setting itopk_size .

Btw, this change fixed the test failures locally for me (went from failing to all passing on my local machine), but I'm still seeing failures in CI - it looks like it passed on the a100/arm64 instance but failed on the amd64 machines =(.

Reply:
Yes, that's why I've asked. It looks like the remaining failing tests are those with `itopk_size` still set.